LorentzVector.hh

```diff
@@ -57,6 +57,38 @@ namespace cola {
             return *this;
         }
 
+        // borrowed from CLHEP. bx by and bz are projections of beta
+        LorentzVectorImpl& boost(Type bx, Type by, Type bz) {
+            Type b2 = bx*bx + by*by + bz*bz;
+            if (b2 >= 1)
+                throw std::runtime_error("Boost faster than speed of light.");
+            Type ggamma = 1.0 / std::sqrt(1.0 - b2);
```

Comment:
will lead to a significant error for HEP calculations as beta2 approx 1

Reply:
Honestly, do we have another choice without using rapidity?

Reply:
nope, so use rapidity)